app/scripts/lib/notification-manager.js

```diff
@@ -1,8 +1,10 @@
-import extension from 'extensionizer'
+import ExtensionPlatform from '../platforms/extension'
 
 const NOTIFICATION_HEIGHT = 620
 const NOTIFICATION_WIDTH = 360
 
+const platform = new ExtensionPlatform()
```

Comment:
Should this be in the constructor?

Reply:
One downside of having it in module scope is that it's eagerly created?

Reply:
That module is just a collection of functions; it doesn't have any state. It shouldn't be a class in the first place really.

I was considering making it no longer a class in a different PR. That would make it easier to get rid of the `platform` global. I can't think of a reason for it to be a class 🤔

Reply:
I guess it couldn't hurt to move it into the constructor in the meantime though. I can do that.